### PR TITLE
[2.13] Backports for 2.13.8.GA

### PR DESCRIPTION
**Pull request by michalvavrik**
### Summary

Backports:

- https://github.com/quarkus-qe/quarkus-test-suite/pull/1048
- https://github.com/quarkus-qe/quarkus-test-suite/pull/1150
- https://github.com/quarkus-qe/quarkus-test-suite/pull/1155
- https://github.com/quarkus-qe/quarkus-test-suite/pull/1261

Please select the relevant options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Dependency update
- [ ] Refactoring
- [x] Backport
- [ ] New scenario (non-breaking change which adds functionality)
- [ ] This change requires a documentation update
- [ ] This change requires execution against OCP (use `run tests` phrase in comment)

### Checklist:
- [ ] Methods and classes used in PR scenarios are meaningful
- [ ] Commits are well encapsulated and follow [the best practices](https://cbea.ms/git-commit/)

### CODE REVIEW

File: spring/spring-web-reactive/src/test/java/io/quarkus/ts/spring/web/reactive/bootstrap/SpringWebQuteReactiveIT.java [hunk 1]
```diff
@@ -1,22 +1,16 @@
...
 import io.quarkus.test.services.Container;
 import io.quarkus.test.services.QuarkusApplication;
+import io.quarkus.ts.spring.web.reactive.MariaDBConstants;
 
 @QuarkusScenario
-@EnabledIfSystemProperty(named = "ts.redhat.registry.enabled", matches = "true")
 public class SpringWebQuteReactiveIT extends AbstractSpringWebQuteReactiveIT {
 
-    @Container(image = IMAGE_105, port = PORT, expectedLog = START_LOG)
+    @Container(image = MariaDBConstants.IMAGE_10, port = MariaDBConstants.PORT, expectedLog = MariaDBConstants.START_LOG_10)
```
Comment (fedinskiy) [inline spring/spring-web-reactive/src/test/java/io/quarkus/ts/spring/web/reactive/bootstrap/SpringWebQuteReactiveIT.java:13]: 105->10?

Comment (fedinskiy) [reply spring/spring-web-reactive/src/test/java/io/quarkus/ts/spring/web/reactive/bootstrap/SpringWebQuteReactiveIT.java:13]: Original file has different change: https://github.com/quarkus-qe/quarkus-test-suite/pull/1155/files#diff-1bc7fd6657bd738a1e60e2d8d0e9cc945fd91e89644cbd1d345afc89a730b09f

Comment (michalvavrik) [reply spring/spring-web-reactive/src/test/java/io/quarkus/ts/spring/web/reactive/bootstrap/SpringWebQuteReactiveIT.java:13]: Original file you pointed at https://github.com/quarkus-qe/quarkus-test-suite/pull/1155/files#diff-1bc7fd6657bd738a1e60e2d8d0e9cc945fd91e89644cbd1d345afc89a730b09f has also 105 -> 10. I believe it is related to the point that we now run this test even when Red Hat registry is not related, as 105 is only available there.

TBH @fedinskiy I didn't do the original chages in #1155 so I can't tell if they were right, but I can tell what this PR does is same as we have on main right now

https://github.com/quarkus-qe/quarkus-test-suite/blob/main/spring/spring-web-reactive/src/test/java/io/quarkus/ts/spring/web/reactive/bootstrap/SpringWebRestReactiveIT.java#L13

and test is passing, therefore I don't see a problem. Please give me more hint if you can see a problem, thanks

Comment (michalvavrik) [reply spring/spring-web-reactive/src/test/java/io/quarkus/ts/spring/web/reactive/bootstrap/SpringWebQuteReactiveIT.java:13]: Although I seem to review that PR, completely forgot.

Comment (rsvoboda) [reply spring/spring-web-reactive/src/test/java/io/quarkus/ts/spring/web/reactive/bootstrap/SpringWebQuteReactiveIT.java:13]: I recall https://github.com/quarkus-qe/quarkus-test-suite/pull/1152

**Pull request merged**